examples/png_example.py

```diff
@@ -1,7 +1,7 @@
 ##############################################################################
 #
 # Example of the PNG image decoder Usage.
-# For dragging to work reasonable, make sure LV_IMG_CACHE_DEF_SIZE is not 0!
+# For dragging to work reasonable, make sure LV_image_CACHE_DEF_SIZE is not 0!
```

Comment:
This is going to be changed again, but current macro name is `LV_CACHE_DEF_SIZE`.
# For dragging to work reasonable, make sure LV_image_CACHE_DEF_SIZE is not 0!
```suggestion
# For dragging to work reasonable, make sure LV_CACHE_DEF_SIZE is not 0!
```

Reply:
Fixed. It was replaced accidentally.